src/azure-cli/azure/cli/command_modules/network/tests/latest/test_nw_flow_log.py

```diff
@@ -9,8 +9,8 @@
 
 class NWFlowLogScenarioTest(ScenarioTest):
```

Comment:
```suggestion
class NWFlowLogScenarioTest(LiveScenarioTest):
```
Could we simply bypass the test cases like above?